.docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Start with a base image containing golang runtime
-FROM golang:1.20
+FROM golang:1.23
```

Comment:
@zond do you anticipate any problems with upgrading to go 1.23? It is required for the new FCM package, or some of its dependencies.